src/Protocol/Protocol.h

```diff
@@ -236,6 +237,8 @@ class cProtocol
 	/** Returns the ServerID used for authentication through session.minecraft.net */
 	virtual AString GetAuthServerID(void) = 0;
 
+	/** Converts eMonsterType to protocol-specific mob types */
+	virtual UInt32 GetProtocolMobType(eMonsterType a_MobType) = 0;
```

Comment:
IMO this shouldn't be public. Consider putting this next to `GetPacketID` in the protected section since they are conceptually similar.